mne/epochs.py

```diff
@@ -1529,7 +1530,7 @@ def as_data_frame(self, picks=None, index=None, scale_time=1e3,
         n_channel_types = 0
         ch_types_used = []
 
-        scalings = _mutable_defaults(('scalings', scalings))[0]
+        scalings = _mutable_default('scalings', scalings)
```

Comment:
I have changed the two places it was used that way, and updated the other dozen or so where it used only a single value